CHANGELOG.next.asciidoc

```diff
@@ -152,6 +153,7 @@ https://github.com/elastic/beats/compare/v8.8.1\...main[Check the HEAD diff]
 
 *Metricbeat*
 
+- Fix `namespace` filter option on metricset `state_namespace` enricher. {pull}39934[39934]
```

Comment:
This should be removed from here as it is inside 'Affecting all beats'

Reply:
https://github.com/elastic/beats/pull/39934#discussion_r1647107974, what do you think? It is not just for filebeat